requirements.txt

```diff
@@ -1,5 +1,5 @@
 torch
 pytorch-ignite
-pytorch-pretrained-bert >= 0.6.2
-tensorboardX
-tensorflow  # for tensorboardX
\ No newline at end of file
+pytorch-transformers>=1.2
+tensorboardX==1.8
+tensorflow  # for tensorboardX
```

Comment:
maybe require `torch>=1.2` so you can use the built-in Tensorboard logger?

Reply:
yeah I've been using torch.utils.tensorboard.SummaryWriter locally. you do still need tensorboard as a dependency, though.

Reply:
ignite v0.2.1 will use the builtin if tensorboardx is not installed (see https://github.com/pytorch/ignite/pull/617). Once that version is released we can delete the tensorboardx requirement.